### DISCUSSION TITLE
Timed depth charges with periscope

### DISCUSSION THREAD

**Discussion by Hazmat616** [FakeFishGames/Barotrauma#12738] [Ideas and suggestions]
When using the periscope have a reticle appear like with other periscopes but can only move in a line. When you fire the depth charge will have a timer set to explode when it would reach the reticle relative position not taking things like currents into account. Using a button wired to the launcher fires as normal so it only sinks. Timed ones will still detonate on collision.

**Comment by Ek-Videogames** [comment]
Minor hiccup with this idea is that periscopes don't see very far, so a 'timed' depth charge fired from the sub would only have a fuse of a second or two at most. You could with sonar terminal, but that'd be more effort to implement code & GUI wise.

**Reply by Hazmat616** [reply]
On a more practical point could you actually be able to hit something off screen if you could set the timer long enough? Otherwise could they increase visibility on it?
